workflows/VGP-assembly-v2/README.md

```diff
@@ -1,28 +1,288 @@
 # Vertebrate Genome Project in Galaxy
 
-## VGP pipeline
+### The Vertebrate Genome Project (VGP)
 
-The VGP pipeline is composed of:
-    
-- Five main workflows
+This repository hosts the Galaxy workflows for the Vertebrate Genome Project's assembly pipeline. Each workflow's *.ga* file can be found in the folder with the corresponding name. For more information about the pipeline in general and the software/technologies used by the VGP for assembly, please see this [full-length GTN training tutorial](https://training.galaxyproject.org/training-material//topics/assembly/tutorials/vgp_genome_assembly/tutorial.html). For a shorter, workflow-focused tutorial, please see this [shortened GTN training tutorial](https://training.galaxyproject.org/training-material/topics/assembly/tutorials/vgp_workflow_training/tutorial.html).
 
-1. Quality control and parameter estimation. It provides a kmer coverage database and an estimation of parameters that will be useful in the genome assembly process.
-2. Phased assembly with Hifiasm
-3. Purging of duplications and overlap from the phased assembly
-4. Scafolding using Bionano optical mapping (optional)
-5. Scafolding using HiC data
-    
-- Two secondary workflows to correct the assembly purging by using custom cutoffs
+## Usage
 
-1. Custom Purging of duplications and overlap from the primary assembly
-2. Custom Purging of duplications and overlap from the alternate assembly
+Workflows currently available on the [Galaxy genome assembly](https://assembly.usegalaxy.eu/) instance.
 
-- Four export workflows to AWS VGP repository
+### Import workflows to Galaxy
 
-1. Phased assembly export
-2. Purged assembly export
-3. Bionano scaffolding export
-4. Hi-C Scafolding export
+> Note : You need to be a registered user in a Galaxy instance to use workflows.
+
+
+To import the VGP workflows into Galaxy, download the `.ga` files on your computer from this github repository.
```

Comment:
For me this step is not easy for the user. What about suggesting to go to [dockstore](https://dockstore.org/organizations/iwc/collections/vgp), click on the workflow they want and use the icon galaxy to import the workflow in their prefered galaxy instance?